.env.sample

```diff
@@ -0,0 +1,4 @@
+export DUNE_USER=
+export DUNE_PASSWORD=
+export DUNE_QUERY_ID=
+export FILE_OUT_PATH=./out
```

Comment:
Unrelated to this PR.

Reply:
True but I'm leaving it here.

Reply:
This is not a blocking issue for me, but I think those things should still be mentioned to foster a better PR etiquette in the long run.

Reply:
Updated PR description.